pkg/cli/tail_test.go

```diff
@@ -27,6 +27,9 @@ import (
 	"google.golang.org/protobuf/types/known/structpb"
 )
 
+// length of `timestamp >= "2023-08-29T17:32:25Z"`.
+const timestampFilterLength = 35
```

Comment:
nit: safer to use len(fmt.Sprintf("timestamp >= %q", time.Now().UTC().Format(time.RFC3339)))? or make the original cutoff global var as well.
var (
  cutoff = "timestamp >= %q"
  timestampFilterLength = len(fmt.Sprintf(cutoff, time.Now().UTC().Format(time.RFC3339)))
)